tests/countries/test_antigua_and_barbuda.py

```diff
@@ -0,0 +1,186 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+from unittest import TestCase
+
+from holidays.countries.antigua_and_barbuda import AntiguaAndBarbuda, AG, ATG
+from tests.common import CommonCountryTests
+
+
+class TestAntiguaAndBarbuda(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(AntiguaAndBarbuda, years=range(1955, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(AntiguaAndBarbuda, AG, ATG)
+
+    def test_new_years_day(self):
+        name = "New Year's Day"
+        self.assertHolidayName(name, (f"{year}-01-01" for year in range(1955, 2050)))
+        dt = (
+            "1995-01-02",
+            "2006-01-02",
+            "2012-01-02",
+            "2017-01-02",
+            "2023-01-02",
+        )
+        observed_name = f"{name} (observed)"
+        self.assertHolidayName(observed_name, dt)
```

Comment:
```suggestion
        self.assertHolidayName(f"{name} (observed)", dt)
```
And same for the rest of such cases below.